.github/workflows/nightly_benchmarks.yml

```
@@ -72,9 +71,9 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.QUTIP_BENCHMARK_S3_SECRET }}
           AWS_EC2_METADATA_DISABLED: true
 
-      - name: run benchmarks
+      - name: Run benchmarks
         run: |
-          python -m qutip_benchmark.cli.run_benchmarks -m nightly -v 
+          python benchmarks/benchmarks.py -v -m nightly
```

```suggestion
          -m qutip_benchmark.cli.run_benchmarks -m nightly -v
```
Perhaps a bad merge?